spec.md

```diff
@@ -534,9 +534,16 @@ Credentials are a set of information used by an application or a user to utilize
 
 #### Log Drain ####
 
-There are a class of service offerings that provide aggregation, indexing, and analysis of log data. To utilize these services an application that generates logs needs information for the location to which it will stream logs. If a broker represents one of these services, it MAY return a `syslog_drain_url` in the response for a request to create a service binding, to which logs MUST be streamed.
+There are a class of service offerings that provide aggregation, indexing, and
+analysis of log data. To utilize these services an application that generates
+logs needs information for the location to which it will stream logs. A
+create binding response from one of these services MUST include a
+`syslog_drain_url`. Applications MUST use this URL when sending logs to the
+service.
```

Comment:
The old text to me was a bit ambiguous as to who is sending the logs, I'll reword it so that we don't imply who is doing the sending.  I assume its ok for the app to send logs there too, right?